server/command/command.go

```diff
@@ -746,7 +770,7 @@ func (r *Runner) actionSelftest(args []string) {
 
 	if len(args) != 2 || args[0] != confirmPrompt || args[1] != "SELF-TEST" {
 		r.postCommandResponse("Are you sure you want to self-test (which will nuke the database and delete all data -- instances, configuration)? " +
-			"All incident data will be lost. To self-test, type `/incident st CONFIRM SELF-TEST`")
+			"All incident data will be lost. To self-test, type `/incident test self CONFIRM SELF-TEST`")
```

Comment:
maybe:

```suggestion
			"All incident data will be lost. To self-test, type `/incident test self CONFIRM TEST SELF`")
```

along with the change the check would require above

Reply:
It makes it more consistent, thanks for the suggestion!

Reply:
Done in https://github.com/mattermost/mattermost-plugin-incident-management/pull/358/commits/8e7adeb91db64c6c7da27e29f94ff891bf5bddaf